EWS subtype updated (#147)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

6. Non-MB and non-ATRT embryonal tumors with <i>CIC-NUTM1</i> or other <i>CIC</i> fusions, were subtyped as `CNS EFT-CIC` (CNS Ewing sarcoma family tumor with <i>CIC</i> alteration).
7. Non-MB and non-ATRT embryonal tumors that did not fit any of the above categories were subtyped as CNS Embryonal, NOS (CNS Embryonal tumor, not otherwise specified).

> Molecular subtype of `EWS` were assigned to patients with _EWSR1_ fusion or with`pathology_diagnosis` of `Ewings Sarcoma`. 
> Additional details can be found in the [analysis README](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/molecular-subtyping-EWS)).
> 

Reviewer: ```suggestion
A molecular subtype of `EWS` was assigned to any tumor with a _EWSR1_ fusion or with a `pathology_diagnosis` of `Ewings Sarcoma`. 
```
